.lando.yml

```diff
@@ -0,0 +1,65 @@
+name: lando-drupal-scaffold
+recipe: drupal8
+config:
+  webroot: web
+
+services:
+  # Use LCM Image.
+  appserver:
+    overrides:
+      services:
+        image: "lastcallmedia/php:7.0-dev"
```

Comment:
Using LCM PHP image; we should talk about what you have/need in that image because we could simplify by piping in some config to the existing lando/php image(s).

But in any event you can swap out for your image if you need to.